composer.json

```diff
@@ -21,16 +21,18 @@
     ],
     "require": {
         "php": ">=5.3.2",
-        "symfony/framework-bundle": "~2.3",
-        "symfony/security-bundle": "~2.3"
+        "symfony/framework-bundle": "~2.6|~3.0",
+        "symfony/http-kernel": "~2.6|~3.0",
+        "symfony/security-bundle": "~2.6|~3.0",
+        "symfony/form": "~2.6|~3.0"
```

Comment:
I will try to fix code to use Symfony components ~2.3 and ~3.0 and i will test everything with Symfony SE >= 2.3